wdl/MergeCohortVcfs.wdl

```diff
@@ -284,44 +308,48 @@ task MakeClusterDupsCombinedBed {
   RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
 
   output {
-    File lookup = "~{cohort}.master_cluster_dups.bed"
-    File cluster_combined="~{cohort}.cluster.combined.bed"
+    File lookup = "~{contig}.master_cluster_dups.bed"
+    File cluster_combined="~{contig}.cluster.combined.bed"
   }
   command <<<
 
     set -euxo pipefail
+    # select rows of BED files pertaining to contig - chrom is 1st column of each BED file
+    awk -F "\t" -v OFS="\t" '{ if ($1 == "~{contig}") { print > "~{contig}.cohort.combined.bed" } }' ~{cohort_combined}
+    awk -F "\t" -v OFS="\t" '{ if ($1 == "~{contig}") { print > "~{contig}.cluster.bed" } }' ~{cohort_cluster}
     python3 <<CODE
-    samp={}
-    var={}
-    with open("~{cohort_combined}","r") as f: # for EACH variant ID, a list of duplicate variants and samples
+    # dictionary of (samples, varIDs) for de-duplicated variant for EACH varID corresponding to that unique variant
+    varID_data = {} 
```

Comment:
Very minor, but in python we usually stick to `lower_camel_case` for variable names, so `varID_data` -> `var_id_data`